spec/features/traject/ead2_indexing_spec.rb

```diff
@@ -1047,4 +1047,15 @@ def all_components(result)
       expect(result).to include("scopecontent_combined_tsm")
     end
   end
+
+  describe "#language_ssm" do
+    let(:fixture_path) do
+      Rails.root.join("spec", "fixtures", "aspace", "generated", "mss", "C0958.processed.EAD.xml")
+    end
+    it "indexes it on the component if appropriate" do
```

Comment:
I'm not sure what "if appropriate" means -- can you clarify the use case?

Reply:
Done